kamon-newrelic/src/main/resources/reference.conf

```diff
@@ -23,6 +23,18 @@ kamon {
 
     # delay between connection attempts to NewRelic collector
     connect-retry-delay = 30 seconds
+
+    subscriptions {
+      trace           = [ "**" ]
+      trace-segment   = [ "**" ]
```

Comment:
I think that trace and trace-segment subscriptions should not be configurable or at least not in the same place. Those two categories are the very foundation of how the New Relic dashboard works and should not be treated the same as we treat custom metrics.

We could always subscribe in code to all traces and segments and change this key to `custom-metric-subscriptions` to hold all the customized additions. That would also feel more natural to the users as they will know for sure that whatever they put in those keys will be reported as custom metrics. 

If you want to go the extra mile, the trace and trace-segment subscriptions could be also configurable via a separate config key, but at the moment I don't find that to be particularly useful.